packages/oc-pages/project_overview/store/modules/deployment_template_updates.js

```diff
@@ -71,6 +71,20 @@ const Serializers = {
         rt.dependencies = rt.dependencies?.filter(dep => {
             return dep.match || dep.target
         })
+
+        rt.properties = rt.properties?.filter(prop => {
```

Comment:
will this filter out properties that are required? pretty sure that will break things